src/main/java/com/kodilla/ecommercee/domain/Order.java

```diff
@@ -34,11 +35,14 @@
     @JoinColumn(name = "user_id")
     @Setter
     private User user;
+
     @OneToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
     @JoinColumn(name = "cart_id")
     @Setter
     private Cart cart;
 
```

Comment:
Pole Cart jest tutaj całkowicie niepotrzebne. 
Pole List< Product > productList zmieniłbym na List< ProductItem> productList oraz zmieniłbym relacje na manytomany.

Reply:
Dodatkowo przerobiłbym Cart  tak aby miał relację manytomany z productItem.

Reply:
Ok , co prawda nie ja dodałem to pole Cart cart do tej klasy ale  naprawie to :dagger: